public/css/remove-data.css

```diff
@@ -0,0 +1,421 @@
+  max-height: 0;
+  opacity: 0;
+  transition: all 0.3s ease;
+}
+
+.relay-toggle-button {
+  background-color: rgba(255, 255, 255, 0);
+  pointer-events: none;
+  transition: all 0.2s ease;
+}
+
+.relay-sign-up-content {
+  transition: all 0.4s ease;
+}
+
+.sending-email {
+  transition: all 0.3s ease;
+}
+
+.sending-email .relay-sign-up-content {
+  filter: blur(0.3px);
+  opacity: 0.5;
+  transition: all 0.4s ease;
+}
+
+.relay-sign-up.sending-email {
+  background-color: #e9e9e9;
+  transition: all 0.3s ease;
+}
+
+.relay-sign-up.sending-email::after {
+  content: "";
+  position: absolute;
+  display: flex;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-image: url("/img/svg/new-loader.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  transition: all 0.3s ease;
+}
+
+.relay-sign-up-confirmation {
+  padding: 24px;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  opacity: 0;
+  visibility: hidden;
+  transition: all 0.3s ease;
+}
+
+.email-sent .relay-sign-up-content {
+  visibility: hidden;
+}
+
+.email-sent .relay-sign-up-confirmation {
+  visibility: visible;
+  opacity: 1;
+  background-color: rgba(255, 255, 255, 1);
+  transition: all 0.3s ease;
+}
+
+.email-sent {
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.email-sent::after {
+  opacity: 0;
+  visibility: hidden;
+  transition: all 0.3s ease;
+}
+
+.relay-confirmation-headline {
+  margin: auto auto 16px auto;
+}
+
+.relay-confirmation-body {
+  margin: 0 auto auto auto;
+  color: #393473;
+  font-size: 18px;
+  line-height: 27px;
+  max-width: 460px;
+}
+
+.relay-confirmation-headline::before {
+  content: "";
+  display: block;
+  background-color: #2ac3a2;
+  border: 3px solid #3fe1b0;
+  background-image: url("/img/svg/white-check.svg");
+  background-size: 50px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  height: 80px;
+  width: 80px;
+  margin: auto auto 16px auto;
+  border-radius: 50%;
+}
+
+@media screen and (max-width: 800px) {
+  .relay-sign-up {
+    flex: 1;
+  }
+
+  .relay-subhead {
+    max-width: 420px;
+  }
+
+  .relay-info {
+    margin: 40px 0;
+  }
+
+  .private-relay-dashboard-card {
+    margin: 12px auto;
+  }
+
+  .private-relay-dashboard-card-content {
+    flex-direction: column;
+    text-align: center;
+    max-width: 400px;
+    margin: auto;
+  }
+
+  .private-relay-dashboard-card-content::before {
+    min-height: 100px;
+    margin-bottom: 16px;
+    margin-right: 0;
+    width: 100%;
+  }
+
+  .private-relay-join-waitlist {
+    margin-top: 16px;
+  }
+
+  .relay-sign-up-wrapper {
+    padding: 44px 24px;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .relay-hero {
+    margin-bottom: 40px;
+  }
+
+  .relay-info {
+    padding: 20px 0;
+    margin: 0;
+  }
+
+  .relay-toggle-button {
+    padding-left: 16px;
+    pointer-events: all;
+  }
+
+  .relay-toggle-button:hover {
+    background-color: rgba(255, 255, 255, 0.1);
```

Comment:
This background appears brighter on mobile. I would recommend going darker instead of lighter. 

```suggestion
    background-color: rgba(0, 0, 0, 0.1);
```

<img width="466" alt="image" src="https://user-images.githubusercontent.com/2692333/113215279-3d5d0280-9240-11eb-8cf1-a8f6ef9a9145.png">